Patches/PlayerControlPatch.cs

```diff
@@ -1310,6 +1310,14 @@ public static void Postfix(PlayerControl __instance, [HarmonyArgument(0)] Player
             PlagueDoctor.OnAnyMurder();
         }
 
+
+        if (Options.GhostAssign(CustomRoles.GuardianAngelTOHE))
+        {
+            target.RpcSetRole(RoleTypes.GuardianAngel);
+            target.RpcSetCustomRole(CustomRoles.GuardianAngelTOHE);
+        }
+            
```

Comment:
Will add a crewmate_check once I'm done with properly setting it up (Ghost roles will be made in such a way that the first x crewmates to die get them)

Reply:
And also neutrals I think, unless I can change impostor team to make guardian angel impostor compatible.